romancal/pipeline/exposure_pipeline.py

```diff
@@ -145,6 +146,9 @@ def process(self, input):
         if not any_saturated:
             self.tweakreg.run(lib)
 
+        # Write out the WfiWcs products
+        self.save_wfiwcs(lib)
```

Comment:
How about making this a step? That way users can skip this, control the output, etc using the existing strun infrastructure?

Reply:
I believe this is an official, archived, output product, much like the resulting models themselves. Hence, if saving output, these should be produced. @schlafly will need to be the final determination.

Reply:
Another thing to note is that currently since this is not a step the romancal version won't be correctly recorded in the metadata (since the saving does not pass the model through [finalize_result](https://github.com/spacetelescope/romancal/blob/06a5c19cd9b24d68967497b206cc9f851b5e10e6/romancal/stpipe/core.py#L76)).

Reply:
so, options are? 

1. call `finalize_result` explicitly
2. make a step
3. add to the return

Methinks the ideal is to add to the return list, but I do not believe that the stpipe internals have been updated for that, i.e., multiple return types.

Reply:
going with calling `finalize_result` explicitly.